x-pack/filebeat/input/s3/s3_integration_test.go

```diff
@@ -245,9 +223,10 @@ func (o *stubOutleter) OnEvent(event beat.Event) bool {
 }
 
 func TestS3Input(t *testing.T) {
-	cfg := defaultTestConfig()
+	cfg := inputConfig()
```

Comment:
I was referring to the name of the `cfg` variable :grimacing:

Reply:
😬Fixed!